File: Semester3/Advanced Methods of Programming/Toy Language Interpreter C#/Assignement7/Assignement7/Model/Command.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Assignement7.Model
{
    public abstract class Command
    {
        private string key;
        private string description;

        public Command(string key, string description)
        {
            this.key = key;
            this.description = description;
        }

        public abstract void Execute();
        public string GetKey() { return key; }
        public string GetDescription() { return description; }
    }
}

File: Semester3/Advanced Methods of Programming/Toy Language Interpreter JavaFx/Assignment8/src/View/Command.java
package View;

public abstract class Command {
    private String key, description;

    public Command(String key, String description){
        this.key = key;
        this.description = description;
    }

    public abstract void execute();

    public String getKey(){
        return key;
    }

    public String getDescription() {
        return description;
    }
}
